IBPSA/Fluid/BaseClasses/MassFlowRateMultiplier.mo

```diff
@@ -0,0 +1,65 @@
+within IBPSA.Fluid.BaseClasses;
+model MassFlowRateMultiplier "Model that multiplies the mass flow rate"
+  extends IBPSA.Fluid.Interfaces.PartialTwoPort;
+
+  parameter Real k "Gain for mass flow rate";
+initial equation
+  assert(k > Modelica.Constants.small or -k < -Modelica.Constants.small,
+    "Gain must not be zero. Received k = " + String(k));
+equation
+    // Pressure drop in design flow direction
+  port_a.p = port_b.p;
+
+  // Mass balance (mass is not conserved by this model!)
+ port_b.m_flow = -k*port_a.m_flow;
+
+  // Specific enthalpy flow rate
+  port_a.h_outflow = inStream(port_b.h_outflow);
+  port_b.h_outflow = inStream(port_a.h_outflow);
```

Comment:
No equations for `allowFlowReversal=false` but that's ok in this case since it's a very specific model.